src/core/Akka.Persistence.TestKit.Tests/MemoryJournalSpec.cs

```diff
@@ -13,9 +14,11 @@ namespace Akka.Persistence.TestKit.Tests
     public class MemoryJournalSpec : JournalSpec
     {
         public MemoryJournalSpec(ITestOutputHelper output)
-            : base(actorSystemName: "MemoryJournalSpec", output: output)
+            : base(typeof(MemoryJournal), "MemoryJournalSpec")
```

Comment:
Missing output propagation to base constructor - this will cause xUnit parallel tests to not print any results.